mittab/apps/tab/judge_views.py

```diff
@@ -212,7 +212,7 @@ def batch_checkin(request):
     round_numbers = list([i + 1 for i in range(TabSettings.get("tot_rounds"))])
     for judge in Judge.objects.all():
         checkins = []
-        for round_number in round_numbers:
+        for round_number in [0] + round_numbers:
```

Comment:
I assume `0` is how you're flagging outs? Keep in mind that when the tournament starts, `cur_round` is = `0`. Just in case that causes any issues somehow

Reply:
I will test to see if there are issues, I don’t think there would be though, since round 0 is never paired, it’s just the default starting value